.travis.yml

```diff
@@ -22,9 +22,9 @@ env:
     # Custom environment variables for the ARM wheel builder
     - CIBW_BUILD_VERBOSITY=1
     - CIBW_TEST_COMMAND="bash {project}/build_tools/travis/test_wheels.sh"
-    - CIBW_ENVIRONMENT="CPU_COUNT=6
-                        OMP_NUM_THREADS=2
-                        OPENBLAS_NUM_THREADS=2
+    - CIBW_ENVIRONMENT="CPU_COUNT=2
+                        OMP_NUM_THREADS=1
+                        OPENBLAS_NUM_THREADS=1
```

Comment:
I think it's good to test code in multithreading mode to be able to detect potential bugs related to multithreading. This should not have any impact on memory usage. `CPU_COUNT` on the other hand should have an impact on memory usage.

```suggestion
                        OMP_NUM_THREADS=2
                        OPENBLAS_NUM_THREADS=2
```